app/institutions/dashboard/-components/departments-panel/component.ts

```diff
@@ -23,6 +23,10 @@ export default class DepartmentsPanel extends Component {
     };
 
     didReceiveAttrs() {
+        if (!this.departments) {
+            return;
+        }
+
         const departmentNumbers = this.departments.map(x => x.numUsers);
         this.chartHoverIndex = departmentNumbers.indexOf(Math.max(...departmentNumbers));
```

Comment:
I think this would be better as:
```ts
        if (this.departments) {
            const departmentNumbers = this.departments.map(x => x.numUsers);
            this.chartHoverIndex = departmentNumbers.indexOf(Math.max(...departmentNumbers));
        }
```

Reply:
Also, I think you need to use `.set()` to make sure the template will respond:
```suggestion
         this.set('chartHoverIndex', departmentNumbers.indexOf(Math.max(...departmentNumbers)));
```